<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>米哥的网络调试助手</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>12</y>
     <width>591</width>
     <height>582</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_all">
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_rev">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>180</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#groupBox_rev{font: 87 10pt &quot;Arial&quot;;}
#groupBox_rev{background-color: rgb(218, 218, 218);}</string>
      </property>
      <property name="title">
       <string>接受区设置</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>117</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBox_1">
          <property name="text">
           <string>接受转向文件...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_2">
          <property name="text">
           <string>自动换行显示</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_3">
          <property name="text">
           <string>显示接受时间</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_4">
          <property name="text">
           <string>十六进制显示</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_5">
          <property name="text">
           <string>暂停接受显示</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>104</width>
         <height>20</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="myLabel" name="label_saveData">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Arial&quot;;
text-decoration: underline;
color: rgb(85, 0, 255);</string>
          </property>
          <property name="text">
           <string>保存数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="myLabel" name="label_clearData">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Arial&quot;;
text-decoration: underline;
color: rgb(85, 0, 255);</string>
          </property>
          <property name="text">
           <string>清除接受</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="4" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QTextEdit" name="textEdit_sendData">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_send">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 170, 0);</string>
          </property>
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QWidget" name="widgetBottom" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>22</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(234, 244, 255);</string>
      </property>
      <widget class="QLabel" name="label_state">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>就绪！</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>0</y>
         <width>261</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_0">
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="label_sendCnt">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>0</y>
               <width>60</width>
               <height>20</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>发送：0</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>5</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>1677215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLabel" name="label_revCnt">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>0</y>
               <width>60</width>
               <height>20</height>
              </rect>
             </property>
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>接受：0</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>5</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_resetCount">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>复位计数</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" rowspan="3">
     <widget class="QGroupBox" name="groupBox_netDataRev">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>430</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#groupBox_netDataRev{font: 87 10pt &quot;Arial&quot;;}
#groupBox_netDataRev{background-color: rgb(218, 218, 218);}</string>
      </property>
      <property name="title">
       <string>网络数据接受</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTextEdit" name="textEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_network">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>190</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#groupBox_network{font: 87 10pt &quot;Arial&quot;;}
#groupBox_network{background-color: rgb(218, 218, 218);}</string>
      </property>
      <property name="title">
       <string>网络设置</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>21</y>
         <width>101</width>
         <height>131</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>(1)协议类型</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_1">
          <item>
           <property name="text">
            <string>UDP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TCP Client</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TCP Server</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>(2)本地主机地址</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>(3)本地主机端口</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="btn_open">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>100</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="0" rowspan="3">
     <widget class="QGroupBox" name="groupBox_send">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>160</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>150</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#groupBox_send{font: 87 10pt &quot;Arial&quot;;}
#groupBox_send{background-color: rgb(218, 218, 218);}</string>
      </property>
      <property name="title">
       <string>发送区设置</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>120</y>
         <width>119</width>
         <height>32</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox_10">
          <property name="text">
           <string>发送周期</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_msData"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>ms</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>104</width>
         <height>16</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Arial&quot;;
text-decoration: underline;
color: rgb(85, 0, 255);</string>
          </property>
          <property name="text">
           <string>文件载入</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">font: 9pt &quot;Arial&quot;;
text-decoration: underline;
color: rgb(85, 0, 255);</string>
          </property>
          <property name="text">
           <string>清除输入</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>21</y>
         <width>121</width>
         <height>101</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="checkBox_6">
          <property name="text">
           <string>启动文件数据源</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_7">
          <property name="text">
           <string>自动发送附加位</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_8">
          <property name="text">
           <string>发送完自动清空</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_9">
          <property name="text">
           <string>按十六进制发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>接受端ip：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_rev"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>端口号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_revHonts"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btn_broadcast">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>广播</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myLabel</class>
   <extends>QLabel</extends>
   <header>mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
